docs/join-the-network/run-avail/other-nodes/0050-avail-bootstrap-node.md

```diff
@@ -0,0 +1,60 @@
+---
+id: avail-bootstrap-node
+title: Run a Bootstrap Node
+sidebar_label: Run a Bootstrap Node
+description: Learn how to run a boostrap node
+keywords:
+  - docs
+  - avail
+  - node
+  - data availability
+  - da
+image: https://availproject.github.io/img/avail/AvailDocs.png
+slug: avail-bootstrap-node
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Bootstrap Node Setup
+
+If for some reason you ever wanted to host your own network, you will need a node that will serve as an access point to all the other nodes wanting to join.
+This is where the Bootstrap node comes onto to the stage. We offer a simple and functioning setup for a such node, just be sure to deploy few of these and that they are publicly accessible over the network.
```

Comment:
typo, maybe rephrase as "we offer a simple functioning bootnode setup" and maybe emphasize the importance of a bootnode i.e their role and why they need to be publicly accessible over a network. Perhaps we can give a guideline on how many bootnodes are sufficient to kickstart the network